import-dashboards.py

```diff
@@ -28,7 +28,6 @@
 NEW_VERSION_FILE = SCRIPT_DIR + '/VERSION'
 OLD_VERSION_FILE = GRAFANA_DB_DIR + '/PERCONA_DASHBOARDS_VERSION'
 HOST             = 'http://127.0.0.1:3000'
-LOGO_FILE        = '/usr/share/pmm-server/landing-page/img/pmm-logo.svg'
```

Comment:
IMHO some changes should be added into the function set_home_dashboard too.

$ cat -n import-dashboards.py | grep LOGO_FILE
    31  LOGO_FILE        = '/usr/share/pmm-server/landing-page/img/pmm-logo.svg'
   405      if os.path.isfile(LOGO_FILE) and os.access(LOGO_FILE, os.R_OK):
   406          print ' * Copying %r to grafana directory %r' % (LOGO_FILE, GRAFANA_IMG_DR)
   407          shutil.copy(LOGO_FILE, GRAFANA_IMG_DR)


Could u create a FB?